packages/driver/src/cy/commands/actions/type.js

```diff
@@ -27,6 +27,7 @@ module.exports = function (Commands, Cypress, cy, state, config) {
       delay: 10,
       release: true,
       parseSpecialCharSequences: true,
+      noUpdate: false,
```

Comment:
this shouldn't be needed since e.g. slate.js is canceling the event somehow, and we should simply respect it and behave as the browser would. I'll take a look at this